gbra/recommender/recommenders.py

```diff
@@ -244,8 +244,13 @@ def _sample_walk_length(self):
         Intuitively, smaller alphas bias towards shorter walks, whereas
         larger alphas bias towards longer walks.
         """
+<<<<<<< HEAD
+        mu = int(round(self._alpha * self._num_steps_in_walk))
```

Comment:
Looks like there's some merge conflicts.  The issues here are 1) I renamed `num_steps_in_walk` to `max_steps_in_walk` and 2) I added a new parameter `beta` to the recommender which is effectively the value of `sigma`.

Reply:
oh i thought i resolved the merge